cmd/soroban-cli/src/commands/network/start.rs

```diff
@@ -0,0 +1,151 @@
+use std::process::{Command, Stdio};
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {}
+
+const FROM_PORT: i32 = 8000;
+const TO_PORT: i32 = 8000;
+const CONTAINER_NAME: &str = "stellar";
+const DOCKER_IMAGE: &str = "stellar/quickstart";
```

Comment:
Many alternatives to docker will require the fully qualified image name:
```suggestion
const DOCKER_IMAGE: &str = "docker.io/stellar/quickstart";
```